src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -14,7 +14,27 @@
 limitations under the License.
 -->
 
-<div layout="row" layout-margin layout-align="center center">
-  Displaying details for replica set {{ctrl.stateParams.replicaSet}} in namespace
-  {{ctrl.stateParams.namespace}}.
+<div layout="column" layout-margin layout-align="center center">
+  Displaying details for replica set {{ctrl.replicaSetDetail.name}} in namespace
+  {{ctrl.replicaSetDetail.namespace}}.
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <th>Pod</th>
+          <th>Start time</th>
+          <th>IP</th>
+          <th>Node</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
+          <td>{{pod.name}}</td>
+          <td>{{pod.startTime}}</td>
+          <td>{{pod.podIP}}</td>
+          <td>{{pod.nodeName}}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
```

Comment:
Yes, agreed. I just dumped the detail information here in this pull request. I didn't want to make the PR too long. In next change I'll actually implement the UI to make it look pretty. Most likely using angular material constructs.